chain/lf3/signer.go

```diff
@@ -0,0 +1,37 @@
+package lf3
+
+import (
+	"context"
+
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-f3/gpbft"
+
+	"github.com/filecoin-project/lotus/api"
+)
+
+type signer struct {
+	wallet api.Wallet
+}
+
+// Signs a message with the secret key corresponding to a public key.
```

Comment:
```suggestion
// Sign signs a message with the secret key corresponding to a public key.
```
Also say something about the implicit assumptions that the public key must be known by wallet and must be BLS?